MathExp/Program.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+
+namespace MathExp
+{
+    class Program
+    {
+        static void Main()
+        {
+            var result = Math.Round((Power(2, 3) + Power(3, 5) * Power(5, 7)), 7);
+            Console.WriteLine($"Значение выражения равно: {result}");
```

Comment:
Вычислять нужно точно, а выводить на консоль можно и приближенное значение.
```suggestion
            var result = (Power(2, 3) + Power(3, 5) * Power(5, 7);
            Console.WriteLine($"Значение выражения равно: {result:F7}");
```